reactive/cuda.sh

```diff
@@ -220,16 +218,9 @@ function install_cuda() {
       return
     fi
 
-    apt-get update -qq
-    # apt-get upgrade -yqq
     # In any case remove nouveau driver
     apt-get remove -yqq --purge libdrm-nouveau*
     # Here we also need to blacklist nouveau
-    apt-get install -yqq --no-install-recommends \
```

Comment:
You're right, I was thinking of a different place in cuda.sh, sorry for the noise.